src/box/iproto.cc

```diff
@@ -2815,9 +2815,16 @@ tx_run_override_triggers(struct event *event, const char *header,
 	event_trigger_iterator_create(&it, event);
 
 	while (event_trigger_iterator_next(&it, &trigger, &name)) {
+		struct mp_ctx mp_ctx_header;
+		mp_ctx_create_default(&mp_ctx_header, iproto_key_translation);
+		struct mp_ctx mp_ctx_body;
+		mp_ctx_create_default(&mp_ctx_body, iproto_key_translation);
+
 		func_adapter_begin(trigger, &ctx);
-		func_adapter_push_msgpack(trigger, &ctx, header, header_end);
-		func_adapter_push_msgpack(trigger, &ctx, body, body_end);
+		func_adapter_push_msgpack(trigger, &ctx, header, header_end,
+					  &mp_ctx_header);
+		func_adapter_push_msgpack(trigger, &ctx, body, body_end,
+					  &mp_ctx_body);
```

Comment:
Lua hashes are used for translation, while `func_adapter` was designed to call arbitrary functions.
Now `func_adapter` is used only for Lua, but we should be ready for new `func_adapter` implementations.
Probably, it will be a good solution to add a used hash function to `struct mp_ctx` in the future - then other languages will use `lua_hash`, because with high probability only Lua has pre-calculated hash on its values.
Let's discuss it @alyapunov before merging this PR.